studio/components/interfaces/Organization/BillingSettingsV2/CostControl/CostControl.tsx

```diff
@@ -110,6 +122,30 @@ const CostControl = ({}: CostControlProps) => {
                 </p>
               )}
 
+              {/** Toggled on spend cap, scheduled change for end-of-cycle */}
+              {subscription?.scheduled_plan_change?.target_plan === 'pro' &&
+                subscription?.scheduled_plan_change?.usage_billing_enabled === false &&
+                subscription?.plan.id === 'pro' && (
+                  <Alert_Shadcn_ className="mb-2">
+                    <IconCalendar className="h-4 w-4" />
+                    <AlertDescription_Shadcn_ className="flex justify-between items-center">
```

Comment:
suss classNames. think these may have been copied and pasted from somewhere.
might not need

Reply:
Pushed up some changes after talking to Joshen